### DISCUSSION TITLE
Unable to create a tunnel between two container instances running on two different VM's hosted on Azure cloud

### DISCUSSION THREAD

**Discussion by hbhokare** [strongswan/strongswan#1078] [Q&A]
We are trying to create IPSec tunnel between two container instances running on two different VMs created inside azure cloud 
environment. We have exact same strongswan configuration on both container instances as mentioned below and observing following issues.

*connections {
 A_B {
version = 2
local_addrs = 20.25.1.19
remote_addrs = 20.24.1.17
mobike=no
local {
auth = psk
id = 20.25.1.19
}
remote {
auth = psk
id = 20.24.1.17
}
children {
A_B {
start_action= trap
esp_proposals = null
}
}
 }
}
secrets {
ike-A_B {
secret = abcd1234
id-1 = 20.25.1.19
id-2 = 20.24.1.17
}
}*

1. Unable to create a tunnel when we initiate tunnel from one of the container instance
2. Able to create tunnel if we initiate tunnel from both the container instances 

Anything are we missing here? Do we have any guidence or recommended configuration setting from strogswan side to make IPsec tunneling working between two container instances running inside two different VM's hosted on cloud.    

Thanks,
-Harish

**Comment by tobiasbrunner** [comment]
> Unable to create a tunnel when we initiate tunnel from one of the container instance

Why not? What's the exact issue? (see #196)
